src/index.js

```diff
@@ -19,6 +20,8 @@ const commands = new Map([
 	['help', help],
 	['', help],
 	['init', init],
+        ['version', version],
```

Comment:
been thinking, how about we tell the version in help itself
@jacoobes and @Allyedge any opinions?

Reply:
A separate command is needed, but we can include version with help command

Reply:
yeah do it